.gitattributes

```diff
@@ -1,10 +1,15 @@
-* text eol=lf
-*.jpg -text
-*.png -text
-*.gif -text
+# Directories
+/.github export-ignore
+/.wordpress-org export-ignore
+/node_modules export-ignore
+/vendor export-ignore
 
-.git                         export-ignore
-.gitignore                   export-ignore
-.gitattributes               export-ignore
-.eslintrc.json               export-ignore
-.stylelintrc.json            export-ignore
+# Files
+/.eslintrc.json export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.stylelintrc.json export-ignore
+/.travis.yml export-ignore
+/composer.json export-ignore
+/composer.lock export-ignore
+/phpcs.xml export-ignore
```

Comment:
I think that the files `CHANGELOG.md`, `package.json`, `package-lock.json` and `READM.md` (we have a separate `readme.text` for wp.org) and the directories `node_modules` and `vendor` should be added as `export-ignore`, right?

Reply:
`node_modules` and `vendor` are already listed above unter _directories_

Changelog is at least not completely useless, but not really necessary either.

Reply:
Added the missing files to the `.gitattributes` file.